sips/minutes/_posts/2016-07-15-sip-minutes.md

```diff
@@ -131,17 +135,17 @@ be resolved before resuming its evaluation.
 revisit the support of async/await in try/catch blocks. When that's considered,
 this SIP should be reopened and and it should see another round of discussion.
 
-#### Proposal SIP-20: Improved lazy val initialization by Dmitry Petrashko
+#### Proposal SIP-20: Improved lazy val initialization presented
 Sebastien reviews the SIP and asks Dmitry, present in the meeting, to correct
 him if he's wrong. He agrees that the SIP is desirable but he's unsure about the
 benchmarks and which of the proposals is faster. Dmitry explains that the
-benchmarks are in the repo, and that in the best case the initialization doesn’t
-need synchronization.
+benchmarks are in the repository.
   
-He also recommends the author to include more documentation.
+Sebastien also recommends the authors to include more documentation.
   
-**Outcome**: Dmitry volunteers to provide an implementation in three months. The
-Process Lead schedules the next iteration by October 2016.
+**Outcome**: Dmitry says he's likely to have time to update the SIP in three
```

Comment:
I'm not the author of original SIP.
The implementation for Dotty is already provided. 
The implementation for Scalac cannot be provided in a minor release without breaking binary compatibility.
I didn't make promise to provide an implementation for Scalac, the promise that I did make is to update the documentation to match the scheme implemented in Dotty. Realistically, I can't promise to implement it for scalac in near future without sacrificing my current Dotty commitments.